doc/adoc/user_guide.adoc

```diff
@@ -302,6 +302,10 @@ a pointer to the respective log file.
   SUSE Linux Enterprise Server documentation.
 * In systems that contain multiple root file systems on different mount points
   only the root file system mounted on `/` (primary system) will be migrated.
+* Upgrade is not supported for systems having the SLE 12 HPC module installed.
+  In SLE 15, HPC is no longer a module but rather a product. As there is not a
+  corresponding SLE 12 HPC product, a migration path from SLE 12 HPC to SLE 15 HPC
```

Comment:
s/SLE 12 HPC/SLE 12/

mentioning what looks like a product name after we state it does not exist is confusing.